pyart/aux_io/arm_vpt.py

```diff
@@ -0,0 +1,351 @@
+        exclude them use the `exclude_fields` parameter. Fields which are
+        mapped by this dictionary will be renamed from key to value.
+    additional_metadata : dict of dicts, optional
+        This parameter is not used, it is included for uniformity.
+    file_field_names : bool, optional
+        True to force the use of the field names from the file in which
+        case the `field_names` parameter is ignored. False will use to
+        `field_names` parameter to rename fields.
+    exclude_fields : list or None, optional
+        List of fields to exclude from the radar object. This is applied
+        after the `file_field_names` and `field_names` parameters.
+
+    Returns
+    -------
+    radar : Radar
+        Radar object.
+    """
+
+    # create metadata retrieval object
+    filemetadata = FileMetadata(
+        'cfradial', field_names, additional_metadata, file_field_names,
+        exclude_fields)
+
+    # read the data
+    ncobj = netCDF4.Dataset(filename)
+    ncvars = ncobj.variables
+
+    # 4.1 Global attribute -> move to metadata dictionary
+    metadata = dict([(k, getattr(ncobj, k)) for k in ncobj.ncattrs()])
+    if 'n_gates_vary' in metadata:
+        metadata['n_gates_vary'] = 'false'  # corrected below
+
+    # 4.2 Dimensions (do nothing)
+
+    # 4.3 Global variable -> move to metadata dictionary
+    if 'volume_number' in ncvars:
+        metadata['volume_number'] = int(ncvars['volume_number'][:])
+    else:
+        metadata['volume_number'] = 0
+
+    global_vars = {'platform_type': 'fixed', 'instrument_type': 'radar',
+                   'primary_axis': 'axis_z'}
+    # ignore time_* global variables, these are calculated from the time
+    # variable when the file is written.
+    for var, default_value in global_vars.iteritems():
+        if var in ncvars:
+            metadata[var] = str(netCDF4.chartostring(ncvars[var][:]))
+        else:
+            metadata[var] = default_value
+
+    # 4.4 coordinate variables -> create attribute dictionaries
+    time = cfradial._ncvar_to_dict(ncvars['time'])
+    _range = cfradial._ncvar_to_dict(ncvars['range'])
+
+    # 4.5 Ray dimension variables
+
+    # 4.6 Location variables -> create attribute dictionaries
+    # the only difference in this section to cfradial.read_cfradial is the
+    # minor variable name differences:
+    # latitude -> lat
+    # longitude -> lon
+    # altitdue -> alt
+    latitude = cfradial._ncvar_to_dict(ncvars['lat'])
+    longitude = cfradial._ncvar_to_dict(ncvars['lon'])
+    altitude = cfradial._ncvar_to_dict(ncvars['alt'])
+    if 'altitude_agl' in ncvars:
+        altitude_agl = cfradial._ncvar_to_dict(ncvars['altitude_agl'])
+    else:
+        altitude_agl = None
+
+    # 4.7 Sweep variables -> create atrribute dictionaries
+    # this is the section that needed the most work since the initial NetCDF
+    # file did not contain any sweep information
+    sweep_number = {
```

Comment:
Use the pyart.common.get_metadata to retrieve a dictionary of metadata that will be consistent and can be customized by the user by adjusting the config file.